gbpservice/nfp/configurator/drivers/firewall/asav/asav_fw_driver.py

```diff
@@ -276,6 +312,14 @@ def configure_interfaces(self, context, resource_data):
             LOG.error(msg)
             raise Exception(msg)
 
+        try:
+            status = self._log_forwarding(mgmt_ip)
+        except Exception:
+            msg = ("Configuration of log forwarding has failed. Reason: %r" %
+                   status.json())
+            LOG.error(msg)
+            return status.json()
```

Comment:
@njagadish   - Can we add retires here ?
For enterprise mode, where visibility is expecting log forwarding to be enable, let say due to some error, log forwarding fails, still we are proceeding and launching the chain.
Is this a valid case ? Is log forwarding a necessary condition for visibility, If yes then we can't proceed as asav vendor will be present only in enterprise mode.